server/inspect.go

```diff
@@ -44,6 +44,7 @@ func (s *Server) getInfo() types.CrioInfo {
 		StorageRoot:       s.config.Config.Root,
 		CgroupDriver:      s.config.Config.CgroupManager,
 		DefaultIDMappings: s.getIDMappingsInfo(),
+		LastReload:        s.lastReload.Unix(),
```

Comment:
Do we still care about cross compilation? there are _windows/_unsupported files, and having this line here will break builds. I am fine with leaving it if we don't care about cross compilation/it doesn't actually break anything

Reply:
/cc @vbatts

Reply:
We did make a big push for windows compiles, but I don't think that is a blocker for us any longer.

Reply:
It will be removed with the next commit. I still do not want to increase technical depth so I guess we should keep a _clean_ separation.